### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by NtshVrm**
Hey, please review the auth functionality. Would be great if you could check the deploy preview for edge cases!
Thanks!

- ### Signup
- ### SignIn _**(also with test credentials)**_
- ### Logout
- ### Alerts for all successes and errors **_(will make toasts later on)_**

### CODE REVIEW

**Review by ayush65 — APPROVED** [review]
Code looks clean 
good folder Structure

File: src/context/auth-context.js [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        firstName: firstname,
+        lastName: lastname,
+      });
+      if (status === 200 || 201) {
+        localStorage.setItem("signup", JSON.stringify({ token: encodedToken }));
+        setToken(encodedToken);
+        localStorage.setItem("user", JSON.stringify({ user: createdUser }));
+        setUser(createdUser);
+        alert("Successfully Created! Proceed to login with your credentials");
+        setTimeout(() => {
```
Comment (Ash-D23) [inline src/context/auth-context.js:32]: don't think there is a need to naviagete using setTimeout. Remove the setTimeout as its already async the function

**Review by Ash-D23 — APPROVED** [review]
